lib/Cake/View/Helper.php

```diff
@@ -546,7 +546,7 @@ protected function _formatAttribute($key, $value, $escape = true) {
  *
  * @param string $message Message to be displayed
  * @param string $okCode Code to be executed after user chose 'OK'
- * @param string $cancelCode Code to be executed after user chose 'Cancel'
+ * @param string $cancelCode Code to be executed after user chose 'Cancel', also executed when okCode doesn't return
```

Comment:
It might cause problems, but it still seems like a bug to me. Why not fix (in `2.9`) it and mention it in the release notes like before (e.g. [2-7-migration-guide](http://book.cakephp.org/2.0/en/appendices/2-7-migration-guide.html))?